src/main/java/de/tum/in/www1/bamboo/server/ServerNotificationTransport.java

```diff
@@ -311,8 +311,8 @@ private JSONObject createJSONObject(Notification notification) {
                                 logErrorToBuildLog("Could not load cached test results!");
                             }
                             logToBuildLog("Loading artifacts for job " + buildResultsSummary.getId());
-                            JSONArray staticAssessmentReports = createStaticAssessmentReportArray(buildResultsSummary.getProducedArtifactLinks(), buildResultsSummary.getId());
-                            jobDetails.put("staticAssessmentReports", staticAssessmentReports);
+                            JSONArray staticCodeAnalysisReports = createStaticCodeAnalysisReportArray(buildResultsSummary.getProducedArtifactLinks(), buildResultsSummary.getId());
+                            jobDetails.put("staticCodeAnalysisReports", staticCodeAnalysisReports);
```

Comment:
Is it fine if you just rename it or do we need to keep a copy available as `staticAssessmentReports` to provide backward compatibility?

Reply:
Static code analysis won't work as long as the change isn't reflected in Artemis. But I think this is fine as the feature is not really used by anyone at the moment. Ideally, we create a new release for Artemis shortly after deploying the plugin.